js/index.js

```diff
@@ -37,6 +37,74 @@ const siteContent = {
   },
 };
 
-// Example: Update the img src for the logo
+// Image SRCs
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+let ctaImg = document.getElementById("cta-img")
+ctaImg.setAttribute('src', siteContent["cta"]["img-src"])
+
+let middle = document.getElementById("middle-img")
+middle.setAttribute('src', siteContent["main-content"]["middle-img-src"])
+
+
+//Navbar content
+let navElement = document.querySelector('nav')
```

Comment:
Instead of getting the main navigation element, we could get `let navItems = document.querySelectorAll('nav a');` and use a forEach for both styling and content manipulation without using `Array.from()` since it's a NodeList:

```
navItems.forEach((link, index) => {
     link.textContent = siteContent['nav'][`nav-item-${index+1}`];
     link.style.color = 'green';
});
```

OR

```
for (let i=0; i < links.length; i++) {
  navItems[i].textContent = siteContent['nav'][`nav-item-${i+1}`];
  navItems[i].style.color = 'green';
};
```